ajax/ajaxVote.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+include('config/init.php');
+
+if(!isset($_SESSION['userID'])) {
+	echoNotLoggedIn("You must be logged in to vote.");
+	die();
+}
+
+$eventID=$_REQUEST['eventID'];
+$sessionUserID=$_SESSION['userID'];
+$direction=$_REQUEST['direction'];
+
+$userVote = doesUserVoteExist($eventID, $sessionUserID);
+
+
+//also assumed 3 cases: 1) insert&vote 2)delete&vote and 3)delete, insert, and vote
+if($userVote == null){
+	insertUserVote($eventID, $sessionUserID, $direction);
+	generalVoteinDB($eventID, $direction);
```

Comment:
Why'd you use the `downVoteInDB` function in the undo endpoint, but here you have a general function? I think these two endpoints might need to be reconciled.